src/pages/CookieClicker.vue

```diff
@@ -1,49 +1,144 @@
 <script setup>
-import { computed, ref } from 'vue';
+import { computed, ref, onUnmounted } from 'vue';
 
 const cookies = ref(0);
+const goldenCookies = ref([]);
+const achievements = ref([]);
+const achievementNotifications = ref([]);
 const buildings = ref([
     { name: 'Cursor', price: 15, cps: 0.1, count: 0},
     { name: 'Grandma', price: 100, cps: 1, count: 0},
     { name: 'Farm', price: 1000, cps: 10, count: 0},
     { name: 'Factory', price: 10_000, cps: 100, count: 0},
 ]);
 
-function buyBuilding(building){
+const upgrades = ref([
+    { name: 'Better Cursors', price: 500, effect: () => buildings.value[0].cps *= 2, bought: false },
```

Comment:
what if I add even cheaper building in the future to be the first building? Always try to make things so if stuff changes in future that you wouldn't need to change much.